twitter-clone/app/src/main/java/io/github/vnicius/twitterclone/data/datasource/usertweets/UserTweetsDataSource.kt

```diff
@@ -50,8 +61,14 @@ class UserTweetsDataSource(
     override fun loadBefore(params: LoadParams<Int>, callback: LoadCallback<Int, Status>) {
         userTweetsDataSourceScope.launch {
             try {
-                val result = userRepository.getUserTweetsAsync(userId, pageSize, --currentPage)
-                callback.onResult(result.toMutableList(), --currentPage)
+                val result = userRepository.getUserTweetsAsync(userId, pageSize, params.key - 1)
+                var nextPage: Int? = params.key - 2
```

Comment:
Do you think it is not better to change the `nextPage` to `previousPage` in this case?